pom.xml

```diff
@@ -53,7 +54,7 @@
         <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
         <java.level>8</java.level>
         <!-- jenkins requirement -->
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.235.5</jenkins.version>

```

Comment:
to keep backward comparability, it is better to only update version in splunk-devops-extend